forest/src/daemon.rs

```diff
@@ -39,7 +39,8 @@ pub(super) async fn start(config: Config) {
         option_env!("FOREST_VERSION").unwrap_or(env!("CARGO_PKG_VERSION"))
     );
 
-    let path: PathBuf = [&config.data_dir, "libp2p"].iter().collect();
+    let mut path: PathBuf = config.data_dir.clone();
```

Comment:
Would it be cleaner to have `let path = config.data_dir.join("libp2p")` or am I missing something?